packages/app-builder-lib/src/util/filename.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-ignore
+import * as _sanitizeFileName from "sanitize-filename"
+import * as path from "path"
+
+export function sanitizeFileName(s: string): string {
```

Comment:
Worth noting that renaming this file may be a breaking change if anybody is relying on `util/sanitizeFileName`.